controllers/moderation.js

```diff
@@ -77,8 +80,37 @@ exports.removedItemListPage = function (aReq, aRes, aNext) {
   // removedItemListQuery
   var removedItemListQuery = Remove.find();
 
+  // removedItemListQuery: byModel
+  if (options.byModel) {
+    modelQuery.findOrDefaultIfNull(removedItemListQuery, 'model', options.byModel, null);
```

Comment:
I also didn't know what to put in the parm list where the `null` is... it does work if I do `/.*/` in [R70 above](https://github.com/OpenUserJs/OpenUserJS.org/pull/700/files#diff-bad434633cfbcb799fd8514a5b741160R70) however it transfers that to the QSP which is probably undesirable which is why I ended up doing the [R84 conditional above](https://github.com/OpenUserJs/OpenUserJS.org/pull/700/files#diff-bad434633cfbcb799fd8514a5b741160R84) to ensure the default to all models. Putting in `'$in [...]'` *(the dot, dot, dot meaning all the model names)* doesn't like the `findOrDefaultIfNull` routine as it puts the single quotes in the condition which can't happen... and of course take them out and the V8 JavaScript debugger throws an exception.